soc/arm/nxp_imx/mcimx8mm_a53/Kconfig.defconfig.mimx8mm_a53

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_MIMX8MM_A53
+
+config SOC
+	default "mimx8mm6"
+
+config FLASH_SIZE
```

Comment:
Would be good for those to be coming in via devicetree and having `zephyr,flash` set.

Reply:
Currently I not have the flash on board, so use fixed value here should be ok? I could update to use your suggested method when we use flash in future.

Reply:
Changed to use device tree.